integration/business-process-tests/tests-integration/bpmn/src/test/java/org/wso2/ei/businessprocess/integration/tests/bpmn/BPMNUserSubstitutionTestCase.java

```diff
@@ -156,7 +159,13 @@ public void testAddSubstitute() throws Exception {
         JSONObject jsonResponse = new JSONObject(getSubResponse);
         Assert.assertTrue("null".equals(jsonResponse.getString("endTime")), "Add sub with endTime null");
         Thread.sleep(2 * 60 * 1000); // need to wait till activation time passed
```

Comment:
Do we need this Thread.sleep after adding awaitality?

Reply:
Fixed. Please refer last commit.